plugins/aea-test-autonomy/aea_test_autonomy/base_test_classes/agents.py

```diff
@@ -102,6 +105,13 @@ class BaseTestEnd2End(AEATestCaseMany, UseFlaskTendermintNode):
     # when you want to run the installation of the packages as part of the test case.
     RUN_AEA_INSTALL = False
 
+    @classmethod
+    def setup_class(cls) -> None:
+        """Setup class"""
+
+        FlaskTendermintDockerImage.use_grpc = cls.USE_GRPC  # wicked
+        super().setup_class()
```

Comment:
this is not great, but we need to change state before the autouse fixture fires. There is a ticket for redesign of `BaseAEATestCase` and `BaseTestEnd2End`, can be addressed then.